src/stories/containers/DraggablePanel.stories.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { css } from "@emotion/react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import { useState } from "react";
+import {
+  DraggablePanel,
+  DraggablePanelCoordinatePair,
+  DraggablePanelProps,
+} from "../../components/containers/DraggablePanel/DraggablePanel";
+import UIThemeProvider from "../../components/theming/UIThemeProvider";
+import { mutedMagenta, gray } from "../../definitions/colors";
+
+export default {
+  title: "Containers/DraggablePanel",
+  component: DraggablePanel,
+} as Meta;
+
+const Template: Story<DraggablePanelProps> = (args) => {
+  const panelDefinitionOjects: DraggablePanelProps[] = [
+    "Panel 1",
+    "Panel 2",
+    "Panel 3",
+    "Panel 4",
+  ].map((panelTitle, panelIndex) => {
+    return {
+      children: () => <p>Panel Contents</p>,
+      panelTitle,
+      isOpen: true,
+      backgroundColorForStorybookOnly: mutedMagenta[`${panelIndex + 1}00`],
+      showPanelTitle: args.showPanelTitle,
+      includeDismissButton: args.includeDismissButton,
+      styleOverrides: args.styleOverrides,
+    };
+  });
+
+  return (
+    <UIThemeProvider
+      theme={{
+        palette: {
+          primary: { hue: gray, level: 500 },
+          secondary: { hue: mutedMagenta, level: 500 },
+        },
+      }}
+    >
+      <StackOrderingKeeper draggablePanelProps={panelDefinitionOjects} />;
+    </UIThemeProvider>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  showPanelTitle: true,
+  includeDismissButton: true,
+  styleOverrides: {},
+};
+
+type StackOrderingKeeperProps = { draggablePanelProps: DraggablePanelProps[] };
+
+function StackOrderingKeeper({
```

Comment:
What are your thoughts on having this be an exported component? Off the top of my head, both the genomics sites and the map may always require having such a component to organize their (possibly) multiple draggable panels. Or do you have another idea for how you'd expect this to be implemented in the map, for example?

Reply:
I definitely thought of that, but I think we'd be better off making that component when we're implementing these `<DraggablePanels />` in the context of an application versus in a more sandbox-y setting like this. But that's just me 😃